plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/internal/common/core/util/CommandLocationLookupStrategy.java

```diff
@@ -29,12 +29,22 @@
 	private static final String LINUX_PATHVAR = "PATH";
 	private static final String WINDOWS_PATHVAR = "Path";
 
+	private static final String LINUX_SEPARATOR = ":";
+	private static final String WIN_SEPARATOR = ";";	
+	
 	public static final CommandLocationLookupStrategy WINDOWS_STRATEGY = 
-			new CommandLocationLookupStrategy(WINDOWS_WHERE, ";", WINDOWS_PATHVAR, new String[]{".exe", ".com"}, null);
+			new CommandLocationLookupStrategy(WINDOWS_WHERE, 
+					// Windows can use either separator or path variable
+					// based on whether eclipse was launched from cygwin or not
+					System.getenv().get(WINDOWS_PATHVAR) == null ? LINUX_SEPARATOR : WIN_SEPARATOR,
```

Comment:
This is not right, even if it path var name is PATH when running from CYGWIN or MinGW bash list separator is still ';' not ':'. Delimiter should be always File.pathSeparator.